src/components/gravidkrav/GravidKrav.tsx

```diff
@@ -64,6 +77,23 @@ export const GravidKrav = (props: GravidKravProps) => {
     });
   };
 
+  const fraDatoValgt = (fraDato: Date) => {
+    if (fraDato) {
```

Comment:
Synes vi burde flytte slik logikk til reduceren for å håndtere "business logikk" og state der.

Reply:
Reduceren håndterer vel ikke side effects?